### DISCUSSION TITLE
How to allow copy / paste in running textual apps

### DISCUSSION THREAD

**Discussion by seebi** [Textualize/textual#4092] [General]
Hi all,

I have observed that it is not possible to copy/paste content in textual applications running in my terminal (macos, iterm2 and default terminal).

I've also read a discussion about allowing this for specific widgets (having dedicated actions and bindings) -> https://github.com/Textualize/textual/discussions/4051

Is there a way to allow this e.g. by disabling the mouse support completely for an app?

best regards

@seebi

**Comment by davep** [comment]
[This FAQ](https://textual.textualize.io/FAQ/#how-can-i-select-and-copy-text-in-a-textual-app) may be helpful here. When it comes to more integrated clipboard support many folk (myself included) have worked with [pyperclip](https://pypi.org/project/pyperclip/) with good results.

**Reply by seebi** [reply]
omg ... sorry that I missed that :-)
